diff --git a/node_modules/hexo-theme-stellar/layout/_partial/widgets/recent.ejs b/node_modules/hexo-theme-stellar/layout/_partial/widgets/recent.ejs
index c54094a..1ff6ffd 100644
--- a/node_modules/hexo-theme-stellar/layout/_partial/widgets/recent.ejs
+++ b/node_modules/hexo-theme-stellar/layout/_partial/widgets/recent.ejs
@@ -27,7 +27,7 @@ function layoutDiv() {
     arr = site.pages.filter(p => p.notebook === page.notebook).sort('-updated')
   } else {
     arr = site.posts.filter( p => p.title && p.title.length > 0)
-    arr = arr.sort("updated", -1)
+    arr = arr.sort("date", -1)
   }
   el += '<div class="widget-body fs14">'
   arr = arr.filter(p => p.title?.length > 0)
diff --git a/node_modules/hexo-theme-stellar/source/css/_components/sidebar/search.styl b/node_modules/hexo-theme-stellar/source/css/_components/sidebar/search.styl
index f61f944..3ee55f3 100644
--- a/node_modules/hexo-theme-stellar/source/css/_components/sidebar/search.styl
+++ b/node_modules/hexo-theme-stellar/source/css/_components/sidebar/search.styl
@@ -21,6 +21,7 @@
     box-sizing: border-box
     font-family: $ff-body
     font-size: $fs-14
+    font-weight: 500
     padding: 12px 0
     color: var(--text)
   .search-button 
diff --git a/node_modules/hexo-theme-stellar/source/css/_custom.styl b/node_modules/hexo-theme-stellar/source/css/_custom.styl
index f429a5c..bee224b 100644
--- a/node_modules/hexo-theme-stellar/source/css/_custom.styl
+++ b/node_modules/hexo-theme-stellar/source/css/_custom.styl
@@ -106,3 +106,6 @@ $boxshadow-card-float = 0 12px 16px -4px rgba(0, 0, 0, 0.2)
 $boxshadow-button = 0 0 2px 0px rgba(0, 0, 0, 0.04), 0 0 8px 0px rgba(0, 0, 0, 0.04)
 $boxshadow-block = 0 1px 4px 0px rgba(0, 0, 0, 0.02), 0 2px 8px 0px rgba(0, 0, 0, 0.02)
 $boxshadow-toast = 0 4px 8px 0px rgba(0, 0, 0, 0.1), 0 12px 16px -4px rgba(0, 0, 0, 0.2)
+
+body
+  font-weight: 500
